lib/ansible/modules/yum_repository.py

```diff
@@ -50,6 +50,11 @@
       - Relative cost of accessing this repository. Useful for weighing one
         repo's packages as greater/less than any other.
     type: str
+  countme:
+    description:
+      - Whether a special flag should be added to a randomly chosen query each week.
+      - Default value is V(false).
+      type: bool
```

Comment:
```suggestion
    description:
      - Whether a special flag should be added to a randomly chosen query each week.
      - Default value is V(false).
    type: bool
```

Reply:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.